plugins/vpp/model/nat/keys_nat.go

```diff
@@ -16,42 +16,21 @@ package nat
 
 const (
 	// natPrefix
-	prefix = "vpp/config/v1/nat"
+	Prefix = "vpp/config/v1/nat"
```

Comment:
This is inconsistent with prefixes from other packages. Prefix should end with `/`. This means that those prefixes below should trim first slash.